prombench/manifests/prombench/benchmark/6_loadgen.yaml

```diff
@@ -48,6 +48,28 @@ data:
         - expr: histogram_quantile(0.99, sum by(path, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(path, method, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(instance, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
+      - name: arthematic_operation
+        interval: 10s
+        type: instant
+        queries:
+        - expr: sum(node_cpu_seconds_total)/sum(container_memory_rss)
```

Comment:
This is not terribly good as a test of operator performance, since it matches one series against one other series, both of which have no labels.

Reply:
So Here for improving the performance i have to use some labels something like this ? 
`sum by (instance)(node_cpu_seconds_total) / sum by (instance)(container_memory_rss)`

Reply:
I have replaced the query with `sum by (container, pod) (rate(container_cpu_usage_seconds_total[5m])) / sum by (container, pod) (container_spec_cpu_quota)`, which calculates the CPU utilization of containers relative to their allocated CPU quota. The query is structured with the labels container and pod.